dockerfile

```diff
@@ -0,0 +1,60 @@
+####################################################
+# To build your own image: 
+#    > docker build --build-arg BUILD_VERSION=0.0.3 --build-arg BUILD_DATE=2019-05-19 -t probot/smee-io:latest .
+#      (Protips: Add an another -t probot/smee-io:0.0.3 for a tag also for the version)
+# To run your image once it's ready:
+#    > docker run -d -p 3000:3000 --name smee-io probot/smee-io:latest
+# To push the image into your docker repository (e.g.):
+#    > docker push probot/smee-io:latest
+# If you wish to remove your dangling images after build, please do the following:
+#    > docker rmi $(docker images -f “dangling=true” -q)
+####################################################
+
+##############################################################################
+# Build arguments
+##############################################################################
+ARG BUILD_VERSION
+ARG BUILD_DATE
+ARG PORT=3000
+
+##############################################################################
+# Build the container with Source code compiled
+##############################################################################
+FROM node:10-alpine as build-env
+## To allow caching before building, we need to remove 'postinstall' from the package.json
+## > See about optimization: https://www.aptible.com/documentation/enclave/tutorials/faq/dockerfile-caching/npm-dockerfile-caching.html
+WORKDIR /source
+ADD . .
+RUN npm install --production --unsafe-perm
+
+##############################################################################
+# Build the final runtime container
+# Note: Port must be > 1024. See https://stackoverflow.com/a/9947222/80527
+##############################################################################
+FROM node:10-alpine
+ARG PORT=3000
+WORKDIR /app/smee-io
+COPY --from=build-env /source .
+RUN addgroup -S probot && \
+    adduser -S -G probot smee-io && \
+    chown -R smee-io:probot /app
+USER smee-io
+
+##############################################################################
+# Define other environment variable if needed.
+##############################################################################
+ENV PORT=$PORT
+
+EXPOSE $PORT
+ENTRYPOINT [ "npm", "start" ]
+
+##############################################################################
+# Label the Image
+##############################################################################
+LABEL name="smee-io" \ 

```

Comment:
I know that doing `\` for multiple lines with one `LABEL` works, but do you mind separating the labels into multiple `LABEL` lines? I find it easier to read, and for metadata that's more important than brevity.

Reply:
Yes, it is ok. I just saw that after docker 1.10 the label thing is not causing anymore issues. 

Ref.: https://docs.docker.com/engine/reference/builder/
> Prior to Docker 1.10, this decreased the size of the final image, but this is no longer the case. You may still choose to specify multiple labels in a single instruction, in one of the following two ways: